dll/win32/comdlg32/comdlg32.manifest

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Look at comctl32 for how to handle these manifest files, you're missing quite a few steps here.